src/system/SystemConfig.h

```diff
@@ -788,3 +788,12 @@ struct LwIPEvent;
 #define CHIP_SYSTEM_CONFIG_USE_ZEPHYR_EVENTFD 0
 #endif
 #endif // CHIP_SYSTEM_CONFIG_USE_ZEPHYR_EVENTFD
+
+/**
+ * @def CHIP_SYSTEM_CONFIG_MAX_LARGE_BUFFER_SIZE_BYTES
+ *
+ * @brief Maximum buffer allocation size of a 'Large' message
```

Comment:
please update a comment to help reading/understanding the code. It is obvious from the name that `CONFIG_MAX_LARGE_BUFFER_SIZE_BYTES` is `configures the maximum size in bytes of a large buffer`.

It should answer things like: what is the buffer/message in this context? where is it used?

Also if this is tied to TCP only this should be explained in the comments.